fast-json-stringify.d.ts

```diff
@@ -0,0 +1,143 @@
+declare module "fast-json-stringify" {
```

Comment:
Remove this declaration. Declare the `build` namespace on the top-level i.e. `declare namespace build { ... }`